src/main/java/data/Camera.java

```diff
@@ -1,14 +1,19 @@
 package data;
 
-        import lombok.EqualsAndHashCode;
-        import lombok.Getter;
-        import lombok.Setter;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
 
 /**
  * Created by Bart.
  * Class to store information about cameras.
  */
+@XmlRootElement(name = "camera")
 @EqualsAndHashCode
+@ToString
```

Comment:
Can some of these annotations be placed on one line (like @EqualsAndHashCode and @ToString)